test.js

```diff
@@ -35,6 +35,106 @@ test('support help shortcut', t => {
 	t.is(cli.help, indentString('\nCLI app helper\n\nunicorn\ncat\n', 2));
 });
 
+test('help with no description', t => {
+	const cli = meow({description: false});
+	t.is(cli.help, '\n');
+});
+
+test('help with a custom message and description', t => {
+	const cli = meow('Unicorn', {description: 'Cat'});
+	t.is(cli.help, '\n  Cat\n\n  Unicorn\n');
+});
+
+test('help with a custom message and no description', t => {
+	const cli = meow('Unicorn', {description: false});
+	t.is(cli.help, '\n  Unicorn\n');
+});
+
+test('disabled help options', t => {
+	const cli = meow('Unicorn', {
+		helpOptions: false,
+		flags: {
+			foo: {type: 'string', description: 'Foo'},
+			bar: {type: 'number', description: 'Bar'}
+		}
+	});
+	t.is(cli.help, '\n  CLI app helper\n\n  Unicorn\n');
+});
+
+test('auto-generated help options', t => {
+	const cli = meow({
+		helpOptions: true,
+		flags: {
+			foo: 'string',
+			output: {
+				type: 'string'
+			},
+			input: {
+				type: 'string',
+				default: 'stdin'
+			},
+			indent: {
+				type: 'number',
+				alias: 'i',
+				default: 2,
+				description: 'Indent level'
+			},
+			enabled: {
+				type: 'boolean',
+				default: false,
+				description: 'Enabled or not'
+			},
+			longWord: {
+				type: 'string',
+				alias: 'lw',
+				default: 'none',
+				description: 'Very very long long word.\nThis is the second line.'
+			}
+		}
+	});
+	t.is(cli.help, `
+  CLI app helper
+
+  Options:
+    --foo <string>
+
+    --output <string>
+
+    --input <string>  (default: stdin)
```

Comment:
[question] Which style do you prefer?

```suggestion
    --input <string>  [default: 'stdin']
```

Reply:
`[default: stdin]`